store/sqlstore/upgrade.go

```diff
@@ -949,3 +951,11 @@ func upgradeDatabaseToVersion530(sqlStore *SqlStore) {
 		saveSchemaVersion(sqlStore, VERSION_5_30_0)
 	}
 }
+
+func upgradeDatabaseToVersion531(sqlStore *SqlStore) {
+	if shouldPerformUpgrade(sqlStore, VERSION_5_30_0, VERSION_5_31_0) {
+		// allow 10 files per post
+		sqlStore.AlterColumnTypeIfExists("Posts", "FileIds", "text", "varchar(300)")
+		saveSchemaVersion(sqlStore, VERSION_5_31_0)
+	}
```

Comment:
We should comment these lines until we release next version:

```suggestion
	// if shouldPerformUpgrade(sqlStore, VERSION_5_30_0, VERSION_5_31_0) {
		// allow 10 files per post
		sqlStore.AlterColumnTypeIfExists("Posts", "FileIds", "text", "varchar(300)")
		// saveSchemaVersion(sqlStore, VERSION_5_31_0)
	// }
```